recipes/msdf-atlas-gen/all/conanfile.py

```diff
@@ -0,0 +1,84 @@
+import os
+from conans import ConanFile, CMake, tools
+
+required_conan_version = ">=1.33.0"
+
+
+class MsdfAtlasGenConan(ConanFile):
+    name = "msdf-atlas-gen"
+    license = "MIT"
+    homepage = "https://github.com/Chlumsky/msdf-atlas-gen"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "MSDF font atlas generator"
+    topics = ("conan", "msdf-atlas-gen", "msdf", "font", "atlas")
+    settings = "os", "compiler", "arch"
+    options = {"fPIC": [True, False]}
+    default_options = {"fPIC": True}
+
+    generators = "cmake"
+    exports_sources = ["CMakeLists.txt"]
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def requirements(self):
+        self.requires("artery-font-format/1.0")
+        self.requires("msdfgen/1.9.1")
+
+    def validate(self):
+        if self.settings.compiler.get_safe("cppstd"):
+            tools.check_min_cppstd(self, 11)
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version],
+                  strip_root=True, destination=self._source_subfolder)
+
+    def _patch_sources(self):
+        cmakelists = os.path.join(
+            self._source_subfolder, "CMakeLists.txt")
+        if not os.path.exists(cmakelists):
+            tools.tools_files.rename("CMakeLists_internal.txt", cmakelists)
+
+        tools.replace_in_file(cmakelists,
+                              "add_subdirectory(msdfgen)", "")
+        tools.replace_in_file(cmakelists,
+                              "target_link_libraries(msdf-atlas-gen PUBLIC Threads::Threads msdfgen::msdfgen msdfgen::msdfgen-ext)",
+                              "target_link_libraries(msdf-atlas-gen PUBLIC Threads::Threads ${CONAN_LIBS})")
+        tools.save_append(cmakelists,
+                          "install(TARGETS msdf-atlas-gen-standalone DESTINATION bin)")
+
+    def _get_or_configure_cmake(self):
+        if self._cmake:
+            return self._cmake
+        self._cmake = CMake(self)
+        self._cmake.definitions["MSDF_ATLAS_GEN_BUILD_STANDALONE"] = True
+        self._cmake.definitions["MSDFGEN_USE_OPENMP"] = False
+        self._cmake.definitions["MSDFGEN_USE_CPP11"] = True
+        # No need to install msdfgen or build its standalone, we just want the msdf-atlas-gen executable
+        self._cmake.definitions["MSDFGEN_INSTALL"] = False
+        self._cmake.definitions["MSDFGEN_BUILD_MSDFGEN_STANDALONE"] = False
```

Comment:
I think these options are used by the vendored `msdfgen`, not by `msdf-atlas-gen`.
```suggestion
        self._cmake.definitions["MSDFGEN_BUILD_MSDFGEN_STANDALONE"] = False
```

Reply:
I added them because they are defined in the [official CMakeLists.txt](https://github.com/Chlumsky/msdf-atlas-gen/blob/master/CMakeLists.txt#L5-L9), but I guess you're right, we don't really care about them here.